includes/footer.php

```diff
@@ -45,7 +44,7 @@
                             <img src="/img/sponsors/FIRSTWa.jpg" alt="/img/sponsors/FIRSTWa.jpg" title="FIRST Washington"/>
```

Comment:
This still looks a little weird...
![FIRST WA](https://cloud.githubusercontent.com/assets/6354860/19276395/15c61c78-8f8b-11e6-9068-2c42df3e087d.PNG)

Also, it might be better to use the horizontal version of the logo. Some versions are available [here](http://firstwa.org/Team-Resources/Team-Communications-Toolkit). Personally, I don't have a preference on which variant to use (tagline or no tagline).

![http://firstwa.org/portals/0/Team%20Resources/Team%20Communications/FirstWashington_logo_horz_RGB_stt_tagline_1024x256_thumb.png?ver=2016-09-27-163612-007](http://firstwa.org/portals/0/Team%20Resources/Team%20Communications/FirstWashington_logo_horz_RGB_stt_tagline_1024x256_thumb.png?ver=2016-09-27-163612-007)![http://firstwa.org/portals/0/Team%20Resources/Team%20Communications/FirstWashington_logo_horz_RGB_1024x256_thumb.png?ver=2016-09-27-162714-303](http://firstwa.org/portals/0/Team%20Resources/Team%20Communications/FirstWashington_logo_horz_RGB_1024x256_thumb.png?ver=2016-09-27-162714-303)